bftengine/src/bftengine/messages/PrePrepareMsg.cpp

```diff
@@ -122,7 +122,8 @@ PrePrepareMsg::PrePrepareMsg(ReplicaId sender,
                              CommitPath firstPath,
                              const concordUtils::SpanContext& spanContext,
                              size_t size)
-    : PrePrepareMsg::PrePrepareMsg(sender, v, s, firstPath, spanContext, std::to_string(s), size) {}
+    : PrePrepareMsg::PrePrepareMsg(
+          sender, v, s, firstPath, spanContext, std::string(1, '0') + std::to_string(s), size) {}
```

Comment:
This is not trivial and needs to be explained in a comment.

Reply:
Thanks @yuliasherman , I will add detailed comment on this.